extensions/blocks/simple-payments/index.js

```diff
@@ -14,6 +14,7 @@ import { getIconColor } from '../../shared/block-icons';
 import edit from './edit';
 import save from './save';
 import deprecatedV1 from './depecrated/v1';
+import deprecatedV2 from './depecrated/v2';
```

Comment:
Maybe we can take this opportunity to:

1. Rename `depecrated/` into `deprecated` to fix this typo.
2. Have a `deprecated/index.js` where we collect all deprecations, a bit like here:
https://github.com/Automattic/jetpack/blob/c3774de77e1c255b7e5e447cbc246dc21daca7e4/extensions/blocks/eventbrite/deprecated/index.js#L2-L7